aiohttp/connector.py

```diff
@@ -467,30 +481,57 @@ def family(self):
         return self._family
 
     @property
+    def cache_dns(self):
```

Comment:
Especially since below you use "cached_hosts" and "clear_dns_cache" terminology.